modules/admin_manual/pages/installation/deployment_recommendations/nfs.adoc

```diff
@@ -0,0 +1,146 @@
+This has become the default on most distributions, but we recommend checking what this is set to in your ownCloud installations.
+
+=== noac
+
+The `noac` option should be used with caution. 
+Quoting the NFS man page:
+
+[quote, nfs man page]
+Using the noac option provides greater cache coherence among NFS clients accessing the same files, *but it extracts a significant performance penalty*. As such, judicious use of file locking is encouraged instead.
+
+== Tune the Read and Write Block Sizes
+
+The allowed block sizes are the packet chunk sizes that NFS uses when reading and writing data.
+The smaller the size, the greater the number of packets need to be sent to send or receive a file.
+Conversely, the larger the size, the fewer the number of packets need to be sent to send or receive a file.
+
+You can find this information by looking at the `/proc/mounts`, as in the example below.
+
+[source,console]
+----
+cat /proc/mounts | grep -i owncloud
+
+192.168.0.103:/data /mnt/owncloud nfs rw,vers=3,rsize=32768,wsize=32768,hard,proto=tcp,timeo=600,retrans=2,sec=sys,addr=192.168.0.103 0 0
+----
+
+Once you've determined the best sizes, set them by passing the (`rsize` and `wsize`) options when mounting the share or in the share's mount configuration.
+
+.Specifying the read and write block sizes when calling mount
+[source, console]
+----
+mount 192.168.0.104:/data  /mnt -o rsize=65536,wsize=65536
+----
+
+=== Tune the MTU (Maximum Transmission Unit) Size
+
+The MTU size dictates the maximum amount of data that can be transferred in one Ethernet frame.
+If the MTU size is too small, then regardless of the read and write block sizes, the data must still be fragmented across multiple frames.
+You can find the current MTU size for each interface using _netstat_, _ifconfig_, _ip_, and _cat_, as in the following examples:
+
+.Retrieve interface MTU size with netstat
+[source,console]
+----
+netstat -i
+
+Kernel Interface table
+Iface      MTU    RX-OK RX-ERR RX-DRP RX-OVR    TX-OK TX-ERR TX-DRP TX-OVR Flg
+lo       65536   363183      0      0 0        363183      0      0      0 LRU
+wlan0     1500  3138292      0      0 0       2049155      0      0      0 BMR
+----
+
+.Retrieve interface MTU size with ifconfig
+[source,console]
+----
+ifconfig| grep -i MTU
+
+lo: flags=73<UP,LOOPBACK,RUNNING>  mtu 65536
+wlan0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
+----
+
+.Retrieve interface MTU size with ip
+[source,console]
+----
+ip addr | grep mtu
+
+1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
+2: wlan0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP group default qlen 1000
+----
+
+.Retrieve interface MTU size with cat
+[source,console]
+----
+cat /sys/class/net/<interface>/mtu
+----
+
+To change the MTU size, under Ubuntu, specify the MTU size, per interface, in `/etc/network/interfaces`, as in the following example, and then restart networking.
```

Comment:
Starting with Ubuntu 18.04 this file/path is not longer used.
The correct file for configuring interfaces is now: `/etc/netplan/50-cloud-init.yaml`
Be adviced, that the content is now in yaml format and needs careful intend formatting !!
You will find examples by searching the internet.

Reply:
Thanks. I wasn't aware of the change.

Reply:
From further reading of the NetworkManager and Netplan documentation, it doesn't appear that setting the MTU is an altogether straight-forward process.